R/s3.put.R

```diff
@@ -11,7 +11,16 @@
 s3.put <- function (x, path, name, bucket.location = "US",
                     debug = FALSE, check_exists = TRUE,
                     num_retries = getOption("s3mpi.num_retries", 0), backoff = 2 ^ seq(2, num_retries + 1),
-                    max_backoff = 128) {
+                    max_backoff = 128, storage_format = "RDS", ...) {
+
+  if (!(storage_format %in% c("RDS", "CSV"))) {
+    stop("The only storage formats supported at the moment are RDS and (for data frames) CSV.")
+  }
+
+  if (identical(storage_format, "CSV") && !is(x, "data.frame")) {
```

Comment:
Use `==` if you have done argument matching / checking. Also use `is.data.frame`